crates/rpc/rpc/src/layers/jwt_secret.rs

```diff
@@ -26,10 +27,6 @@ pub enum JwtError {
     MissingOrInvalidAuthorizationHeader,
     #[error("JWT decoding error {0}")]
     JwtDecodingError(String),
-    #[error("IO error occurred while reading {path}: {err}")]
-    IORead { err: std::io::Error, path: PathBuf },
-    #[error("IO error occurred while writing {path}: {err}")]
-    IOWrite { err: std::io::Error, path: PathBuf },
     #[error("An I/O error occurred: {0}")]
     IOError(#[from] std::io::Error),
```

Comment:
this should now be an `FsPathError` error I think

I'd like to not use eyre here

Reply:
Yes I understand, less prone to error, I just changed that.